src/components/Toast/Toast.module.css

```diff
@@ -1,170 +1,57 @@
-@import '../../design-tokens/mixins.css';
-
+@import '../../design-tokens/mixins.css'
```

Comment:
```suggestion
@import '../../design-tokens/mixins.css';
```

? not sure if it makes a difference

Reply:
Holy crap it was the semicolon, thanks